### PR TITLE
Feat: add types

### PR DESCRIPTION
**Pull request by javierlinked**

### CODE REVIEW

**Comment by shystruk** [comment]
Hi @javierlinked 
Can you please add a demo page with .ts where we can test that the package works.
You can add a new .html page to **demo** folder or create a new **demo-ts** folder :) 
Thanks

**Comment by javierlinked** [comment]
Hello @shystruk. Done :D

File: demo-ts/index.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
+import CSVFileValidator from '../src/csv-file-validator'
```
Comment (shystruk) [inline demo-ts/index.js:1]: @javierlinked Can you please create the index.ts file and make sure you import .ts  module to make sure type definitions work well.

**Review by shystruk — CHANGES_REQUESTED** [review]

**Comment by shystruk** [comment]
@javierlinked thank you for the contribution. In #79 type definitions were added.

**Pull request closed without merging**